pwndbg/aglib/elf.py

```diff
@@ -202,6 +202,14 @@ def get_containing_sections(elf_filepath: str, elf_loadaddr: int, vaddr: int):
     return sections
 
 
+def get_vmlinux_unrand_base(elf_filepath: str):
+    elf = get_elf_info(elf_filepath)
```

Comment:
```suggestion
    # This should work fine for any standard vmlinux output
    # but may not work for *any* ELF file as it may have the first
    # PT_LOAD with offset != 0
    # See: https://github.com/pwndbg/pwndbg/pull/3269#discussion_r2299313293
    elf = get_elf_info(elf_filepath)
```

Reply:
Lol I can't commit this suggestion :D